galeracluster/source/error.rst

```diff
@@ -0,0 +1,62 @@
+========================
+Server Error Log
+========================
+.. _`server-error-log`:
+
+
+
+.. rubric:: ``Node 0 (XXX) requested state transfer from '*any*'. Selected 1 (XXX) as donor.``
+.. _`node-requested-sst-from-any`:
+
+The node is attempting to initiate a :term:`State Snapshot Transfer`.
+
+In the event that you do not explicitly set the donor node through :ref:`wsrep_sst_donor <wsrep_sst_donor>`, the Group Communication module selects a donor based on the information available about the node states.
+
+Group Communication monitors node states for the purposes of flow control, state transfers and quorum calculations.  That is, to ensure that a node that shows as ``JOINING`` does not count towards flow control and quorum.
+
+The node is considered fit to serve as a donor if it returns a ``SYNCED`` state.  The specific donor is chosen as first among the available synced nodes in the index.  When the node is chosen its state changes immediately to ``DONOR``, meaning that it is no longer available for requests.
+
```

Comment:
When selecting a donor, preference is given to nodes having the same gmcast.segment wsrep provider option as the joining node.